website/addons/dropbox/utils.py

```diff
@@ -43,10 +43,14 @@ def log(self, action, extra=None, save=False):
         :param dict extra: Extra parameters to add to the ``params`` dict of the
             new NodeLog.
         """
+        if self.node.get_addon('dropbox', deleted=True).folder == '/':
+            folder = '/ Full DropBox'
```

Comment:
In other words, store `/` in the the log record in the database but show `/ (Full Dropbox)` on the front-end.